phpmailer/phpmailer.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * Name: PHP Mailer SMTP
+ * Description: Connects to a SMTP server based on the config
+ * Version: 0.1
+ * Author: Marcus Mueller <http://mat.exon.name>
+ */
+
+use Friendica\App;
+use Friendica\Core\Addon;
+use PHPMailer\PHPMailer\PHPMailer;
+use PHPMailer\PHPMailer\Exception;
+
+function phpmailer_install()
+{
+	Addon::registerHook(
+		'emailer_send_prepare',
+		'addon/phpmailer/phpmailer.php',
+		'phpmailer_emailer_send_prepare'
+	);
+}
+
+function phpmailer_uninstall()
+{
+	Addon::unregisterHook(
+		'emailer_send_prepare',
+		'addon/phpmailer/phpmailer.php',
+		'phpmailer_emailer_send_prepare'
+	);
+}
+
+function phpmailer_module()
+{
+}
+
+/**
+ * @param App $a
+ * @param array $b
+ */
+function phpmailer_emailer_send_prepare(App $a, array &$b)
+{
+	require_once __DIR__ . '/phpmailer/src/PHPMailer.php';
+	require_once __DIR__ . '/phpmailer/src/SMTP.php';
+	require_once __DIR__ . '/phpmailer/src/Exception.php';
+
+	// Passing `true` enables exceptions
+	$mail = new PHPMailer(true);
+	try {
+		if (!empty($a->config['system']['smtp']) && (bool)$a->config['system']['smtp'] === true) {
```

Comment:
This can be simplified to `if (Config::get('phpmailer', 'enabled')) {`

Reply:
Changed to `if (Config::get('phpmailer', 'smtp')) {`